src/lib/backend/PostgreSQLDatabaseService.ts

```
@@ -0,0 +1,90 @@
+                return dataSource;
+            } catch (error) {
+                console.error("[PostgreSQL] Connection error:", error);
+                dataSource = undefined;
+                connectPromise = undefined;
+                throw error;
+            }
+        })();
+
+        return connectPromise;
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)